types/annotations.go

```diff
@@ -0,0 +1,23 @@
+// Copyright 2021 cli-docs-tool authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package types
+
+const (
+	// AnnotationExternalUrl specifies an external link annotation
+	AnnotationExternalUrl = "docs.external.url"
```

Comment:
I'm not always a fan of a blanket "types" package; similar to a "utils" package; it easily becomes a bit of a kitchen-sink of things.

Would an `annotation` package work for you? (`annotation.ExternalURL` etc)

Reply:
(Could be a subdir under "types" if you prefer that)

Reply:
(Oh; so yes, URL should be all-caps for proper camelCase)

Reply:
done